ops/terraform/modules/reporting/reporting.tf

```diff
@@ -0,0 +1,99 @@
+// Reporting module to be used in each AoU deployment (environment). The local "deployment"
+// shares a project with "test", so it's not 1:1.
+
+// Module-specific variables here.
+variable reporting_dataset_id {
+  description = "BigQuery dataset for workbench reporting data."
+  type        = string
+}
+
+locals {
+  //  The module path is in a hidden directory under the running directory of the calling module, but
+  // our view files and schemas for BigQuery are in whatever directory this file is located, and apparently
+  // don't get loaded into that dir by default.
+  // FIXME(jaycarlton): package these files as what Java would call resources, or otherwise find the
+  //  right way to do it. path.module and path. At the vary least, it should work for other user names.
+  //  There has to be a way to separate these cleanly; otherewise you'd have the constraint that everything
+  //  lives in one repo.
+
+  schemas_dir = "/Users/jaycarlton/repos/workbench/ops/terraform/modules/reporting/schemas"
```

Comment:
This should be fixed before merging. One low-tech solution could be to make this a variable that must be provided by the user, then provide instructions on how to populate this.

Reply:
Yes, definitely. I *think* the way I'm supposed to do it is to deploy these files so that they become a resource and then use their paths or contents that way.